packages/styles/src/components/header/_mixins.scss

```
@@ -95,7 +96,8 @@ $interactive-elements-selector: 'a, button, [role="link"], [role="button"]';
   }
 
   @include utilities.focus-style-custom {
-    border-radius: 0.5rem;
+    outline-offset: calc(-2 * #{tokens.get('focus-outline-width', helpers.$post-focus)}) !important;
```

Multiplying is probably not the thing we actually want to do here. It's also a good idea to specify the unit.

```suggestion
    outline-offset: calc(-2px - #{tokens.get('focus-outline-width', helpers.$post-focus)}) !important;
```